.github/policies/breaking-change.yml

```diff
@@ -0,0 +1,46 @@
+# Automation in the `/.github/policies` folder is processed by the dotnet-policy-service app,
+# operated by the [microsoft1estools](https://github.com/microsoft1estools) team.
+# See https://github.com/apps/dotnet-policy-service for more info.
+id: breaking-change
+name: breaking-change
+description: When an issue or PR is labeled as a breaking change, add the label that a breaking change doc is needed, and comment with the breaking change process.
+owner: 
+resource: repository
+disabled: false
+where: 
+configuration:
+  resourceManagementConfiguration:
+    eventResponderTasks:
+    - if:
+      - payloadType: Issues
+      - labelAdded:
+          label: breaking-change
+      then:
+      - addReply:
+          reply: >-
+            Refer to the [.NET SDK breaking change guidelines](https://github.com/dotnet/sdk/blob/main/documentation/project-docs/breaking-change-guidelines.md#required-process-for-all-net-sdk-breaking-changes)
```

Comment:
Linking to SDK repo.

Reply:
The SDK repo hosts the tooling breaking change guidance doc. I don't think we'd want multiple copies as we'd have to change them all whenever there were updates and SDK seems as good a central location as any.